Doc/whatsnew/3.10.rst

```diff
@@ -109,6 +109,12 @@ base64
 Add :func:`base64.b32hexencode` and :func:`base64.b32hexdecode` to support the
 Base32 Encoding with Extended Hex Alphabet.
 
+codecs
+------
+
+Add a :func:`codecs.unregister` to unregister a codec search function.
```

Comment:
Please add the word "function" after `codecs.unregister`.

Reply:
Thanks, updated.